cmd/juju/commands/bootstrap.go

```diff
@@ -145,6 +167,12 @@ Examples:
     juju bootstrap --agent-version=2.2.4 aws joe-us-east-1
     juju bootstrap --config bootstrap-timeout=1200 azure joe-eastus
 
+    # For a bootstrap on k8s, setting the service type of the Juju controller service to LoadBalancer
+    juju bootstrap --config controller-service-type=LoadBalancer
```

Comment:
I tried to do this, and I got "unknown controller-service-type LoadBalancer" it seems it actually needs to be "loadbalancer" one word, but all lowercase.

Reply:
Fixed